### PR TITLE
Create nigelchike

### PR DESCRIPTION
**Pull request by nigelchike**
Add my location to pin

### CODE REVIEW

**Comment by shajanjp** [comment]
Your pin should be in this format.
```
---
githubHandle: <YOUR_USER_NAME>
latitude: <LATITUDE>
longitude: <LONGITUDE>
---
```
@nigelchike

File: _pins/nigelchike [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: nigelchike
+Latitude:39.0840
+Longitude:-76.7002
```
Comment (hectorsector) [inline _pins/nigelchike]: @nigelchike 
Great work so far! You'll need to make sure that your code matches exactly the template in the class, including `---` before and after the code:
```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```

File: _pins/nigelchike2 [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: nigelchike
+Latitude: 39.0840
+Longitude: -76.7002
+---
```
Comment (hectorsector) [inline _pins/nigelchike2]: @nigelchike you can remove this file altogether -- let me know if you need any help with that!

File: _pins/nigelchike [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: nigelchike
+Latitude: 39.0840
+Longitude: -76.7002
```
Comment (hectorsector) [inline _pins/nigelchike]: @nigelchike just use lowercase `latitude` and `longitude`, and add a `.json` extension to the filename and you should be all set!

File: _pins/nigelchike2 (hunk 2 above)
Comment (nigelchike) [inline _pins/nigelchike2]: Thanks @hectorsector. I do need help with removing this file. I also need help adding a json extension to my original file.

**Comment by hectorsector** [comment]
Hey it looks like you figured it out! 

Eagerly awaiting test results!

**Comment by hectorsector** [comment]
We're all green, @nigelchike! Merge when you're ready.

**Comment by nigelchike** [comment]
Thanks again @hectorsector.

**Pull request merged**